qualification_task_behavior_tree.py

```diff
@@ -0,0 +1,96 @@
+""" Script with implementation of behavior tree designed for qualification task (yellow gate)
+
+"""
```

Comment:
Not sure if you have to add blank line to docstring if you don't add description. Look it up in PEP and just try if it works with single line docstring in VSC hinting